cases/templates/cases/add/index.html

```diff
@@ -34,6 +34,15 @@ <h1 class="govuk-heading-l">{{ wizard.form.title }}</h1>
     radius.forEach(function(r) {
         r.addEventListener('change', nw.react);
     });
+    const hasReviewDate = document.querySelector('#id_internal-flags-has_review_date');
+    const reviewDate = document.querySelector('#id_internal-flags-review_date');
+    let f = function() {
+        reviewDate.hidden = !hasReviewDate.checked;
+    };
+    hasReviewDate.addEventListener('change', function() {
```

Comment:
Ditto (although here I think you need some guards because presumably hasReviewDate and reviewDate will not always be present on the page and this will then error)